docs/pages/enroll-resources/desktop-access/troubleshooting.mdx

```diff
@@ -311,10 +311,10 @@ why the certificate cannot be verified.
 
 ### RDP server only uses Standard RDP Security
 
-Attempts to connect to a desktop fail, and the logs show an error similar to:
+Attempts to connect to a desktop fail with an error similar to
```

Comment:
```suggestion
Attempts to connect to a desktop fail with an error similar to:
```
Nit: I would restore the colon since the example is on a separate line.

Reply:
Sorry, I forgot to do this before I turned on auto merge.